dimod/bqm/adjmapbqm.pxd

```diff
@@ -28,15 +28,63 @@ ctypedef map[VarIndex, Bias].const_iterator NeighborIterator
 
 
 cdef class cyAdjMapBQM:
+    """A binary quadratic model where the neighborhoods are c++ maps.
```

Comment:
Maybe something like "A binary quadratic model implemented with c++ maps for variable neighborhoods." ?

Reply:
I'd like to keep it under 80 character width for the first line of the docstring